Documentation/CRDs/Block-Storage/ceph-block-pool-rados-namespace-crd.md

```diff
@@ -49,3 +49,32 @@ If any setting is unspecified, a suitable default will be used automatically.
 ### Spec
 
 - `blockPoolName`: The metadata name of the CephBlockPool CR where the rados namespace will be created.
+
+## Creating a Storage Class
+
+Once the RADOS namespace is created, an RBD-based StorageClass can be created to
+create PVs in this RADOS namespace. For this purpose, the `clusterID` value from the
+CephBlockPoolRadosNamespace status needs to be put into the `clusterID` field of the StorageClass
+spec. 
+
+Extract the clusterID from the CephBlockPoolRadosNamespace CR:
+
+```console
+$ kubectl -n rook-ceph  get cephblockpoolradosnamespace/namespace-a -o jsonpath='{.status.info.clusterID}'
+80fc4f4bacc064be641633e6ed25ba7e
+```
+
+In this example, replace `namespace-a` by the actual name of the radosnamespace
+created before
+Now set the `clusterID` retrieved from the previous step into the `clusterID` of the storage class.
+Example:
```

Comment:
Is it your intent to have "Example:" on a new line? If so, by markdown syntax rules, there needs to be an empty line between "Now set..." and "Example:". Otherwise, "Example:" will appear at the end of the previous sentence when rendered.

I usually use VSCodes markdown preview mode when writing docs so I can be sure the basic rendering shows what I intend.